css-positioning/fixed.css

```diff
@@ -0,0 +1,13 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+  position: fixed;
+  right: 480px;
```

Comment:
This is a very large measurement, which will cause problems when you start to work with responsive CSS. Instead, you should set the `width` of the element to `100%` so that the `text-align` property can do the work of centering the text.